core/tm/schemas/disclaimer.ts

```diff
@@ -0,0 +1,11 @@
+import * as mongoose from 'mongoose';
+import { AuditPlugin } from '@andes/mongoose-plugin-audit';
+
+const DisclaimerSchema = new mongoose.Schema({
+    version: String,
+    texto: String,
+    activo: Boolean
+});
+
+DisclaimerSchema.plugin(AuditPlugin);
+export const Disclaimer = mongoose.model('disclaimer', DisclaimerSchema, 'disclaimer');
```

Comment:
Yo le pondría a la collección `AuthDisclaimers` así quedan agrupadas con las de Auth. Cuando uno trabaja en la DB muchas veces se encuentra con tablas que no saben donde se usan. Es una apreciación mía, otro más puede opinar al respecto. 

```suggestion
export const Disclaimer = mongoose.model('disclaimer', DisclaimerSchema, 'AuthDisclaimers');
```

Reply:
Se modifica el nombre de la colección.